### ISSUE TITLE
stream support?

### ISSUE THREAD

**Issue by daveajones** [bipa-app/lnd-rs#18]
I got this library working well in my code.  Do you have support for grpc streams yet?  If not, I'd be happy to add that over time and submit a PR if that's an option.

**Comment by lsunsi** [comment]
@daveajones I didn't add the support for streams yet, but it's definitely something we'd like to have. I'll revise any PRs you have in this regard and I'll add this kind of support myself further down the line as well.

**Comment by daveajones** [comment]
10-4. I’m also adding some missing calls (like list_invoices) to the published code. Is that something you have already done in an unpublished version or is it ok to submit those changes as well?  I’m assuming this stuff should be held in a “dev” branch when submitting?

**Comment by lsunsi** [comment]
Everything that's currently done is in the repo, there's no unpublished code as of now. Feel free to submit all improvements including the list_invoices and other calls that are missing! Since there's no release done, and the releases will be controlled on tags, you can pull request into master no problem.

By the way, you're using the lib directly from GitHub, right? Would you benefit from a cargo release? I could do it when we get some more things in order.

**Comment by daveajones** [comment]
Yes, I’m using it from the GitHub source. I’m not in a rush for a cargo package so I’m cool with waiting until it has a bit more of the api calls exposed.
